test/functional/wallet_listreceivedby.py

```diff
@@ -18,8 +21,7 @@ def set_test_params(self):
     def run_test(self):
         # Generate block to get out of IBD
         self.nodes[0].generate(1)
-
-        self.log.info("listreceivedbyaddress Test")
```

Comment:
No need to remove this. It looks like this log indicates that this part of the test is for `listreceivedbyaddress` compared with the next parts of the test for `getreceivedbyaddress` and `listreceivedbylabel`/`getreceivedbylabel`.

Reply:
Thanks. Removing the log statement was incorrect. Fixed up...